.github/workflows/refresh_pga_data_function.yml

```diff
@@ -19,9 +35,35 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r gcp/cloud-functions/refresh-pga-data/requirements.txt
+    - name: Wait for Postgres to be ready
+      run: |
+        until pg_isready -h localhost -p 5432; do
+          echo "Waiting for Postgre to be ready..."
+          sleep 2
+        done
+    - name: Install Sqitch
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y sqitch
+    - name: Apply Sqitch migrations
+      working-directory: gcp/migrations
+      env:
+        PGUSER: postgres
+        PGPASSWORD: postgres
+        PGHOST: localhost
+        PGPORT: 5432
+        PGDATABASE: postgres
+      run: |
+        sqitch deploy db:pg://$PGUSER:$PGPASSWORD@$PGHOST:$PGPORT/$PGDATABASE
     - name: Test with pytest
```

Comment:
in a follow up PR i'll add some integration tests that actually use the DB, but this MR just sets everything up